modules/client-configuration/pages/channels.adoc

```diff
@@ -42,3 +42,13 @@ In most cases, you will need to install the base channel for your distribution,
 
 If you need to transform Appstream repositories into non-modular repositories for use with some upgrade operations, you can use the Appstream filter with the content lifecycle management tool.
 For more information about the Appstream filter, see xref:administration:content-lifecycle-examples.adoc[].
+
+
+
+== Updates Channels on {sle} Clients
```

Comment:
There is an important difference between the normal update channel and the "installer update channel".

Update channel exist for all products.

New are the Installer Updates Channel which contain only updates for the  OS "installer" (yast, libzypp, curl, etc).
These channels require in SCC no credentials and are used to update the unified installer media before it installes the OS.
All packages which are in Installer Updates Channel are also in the normal Update Channel. But the normal Update Channel can only be used after the product is registered. Installer Update Channels can be used also before registration and therefore can also fix bugs in the registration code itself.

We need to make it clear. This section is for the Installer Updates.

Reply:
ah ha, that makes sense. i'll edit.